packages/theme/src/theme/_navs.scss

```diff
@@ -102,33 +102,34 @@
 
 			> a,
 			> button {
-				padding: $padding-normal;
-				letter-spacing: 0.6px;
-				line-height: 1;
-				text-transform: uppercase;
-				color: rgba(1, 1, 1, 0.6);
 				background: transparent;
 				border: none;
+				border-radius: 0;
+				color: $dark-silver;
+				letter-spacing: 0.6px;
+				line-height: 1;
+				margin-bottom: 0.5rem;
 				outline: none;
+				padding: $padding-normal;
+				text-transform: uppercase;
```

Comment:
Is that text always supposed to be upper cased?

Reply:
That's what is described in ux guideline